lib/src/test/kotlin/starknet/crypto/StarknetCurveTest.kt

```diff
@@ -94,6 +94,13 @@ internal class StarknetCurveTest {
         assertEquals(PUBLIC_KEY, publicKey)
     }
 
+    @Test
+    fun getPublicKeyInvalid() {
```

Comment:
Nitpick: you could wrap the test name in backticks and add spaces around the words.

Reply:
Yup, but I just wanted to conform to the names that are already in this file